reference-architecture/azure-ansible/master.json

```diff
@@ -0,0 +1,261 @@
+            },
+            {
+              "caching" : "None",
+              "createOption" : "Empty",
+              "diskSizeGB" : "128",
+              "lun" : 1,
+              "name" : "datadisk2",
+              "vhd" : {
+                "uri" : "[concat('http://', parameters('sa'), '.blob.core.windows.net/openshiftvmachines/', parameters('vmName'), '_datadisk2.vhd')]"
+              }
+            }
+          ]
+        },
+        "networkProfile" : {
+          "networkInterfaces" : [
+            {
+              "id" : "[resourceId('Microsoft.Network/networkInterfaces', concat(parameters('vmName'), 'nic'))]"
+            }
+          ]
+        }
+      }
+    },
+    {
+      "type" : "Microsoft.Compute/virtualMachines/extensions",
+      "name" : "[concat(parameters('vmName'),'/installcustomscript')]",
+      "apiVersion" : "[variables('apiVersion')]",
+      "location" : "[resourceGroup().location]",
+      "tags" : {
+        "displayName" : "VirtualMachineCustomScriptExtension"
+      },
+      "dependsOn" : [
+        "[concat('Microsoft.Compute/virtualMachines/', parameters('vmName'))]"
+      ],
+      "properties" : {
+        "publisher" : "Microsoft.OSTCExtensions",
+        "type" : "CustomScriptForLinux",
+        "typeHandlerVersion" : "1.3",
+        "settings" : {
+          "fileUris" : [
+            "[concat(parameters('baseTemplateUrl'), 'master.sh')]"
+          ],
+          "commandToExecute" : "[ concat('sh master.sh ', parameters('adminUsername'), ' ', parameters('adminPassword'), ' ', reference(concat(parameters('vmName'), 'pip')).dnsSettings.fqdn, ' ', padLeft(parameters('numberOfNodes'), 2, '0'), ' ', parameters('routerExtIP'),' ', parameters('RHNUserName'), ' ', parameters('RHNPassword'),' ', parameters('SubscriptionPoolId'), ' ',  variables('sQuote'),  parameters('sshPrivateData'), variables('sQuote'),  ' ', variables('sQuote'), parameters('sshKeyData'), variables('sQuote')  ) ]"
+        }
+      }
+    },
+    {
+      "type" : "Microsoft.Network/networkSecurityGroups",
+      "name" : "[concat(parameters('vmName'), 'nsg')]",
+      "tags" : {
+        "displayName" : "NetworkSecurityGroup"
+      },
+      "apiVersion" : "[variables('apiVersion')]",
+      "location" : "[resourceGroup().location]",
+      "properties" : {
+        "securityRules" : [
+          {
+            "name" : "default-allow-ssh",
+            "properties" : {
+              "protocol" : "Tcp",
+              "sourcePortRange" : "*",
+              "destinationPortRange" : "22",
+              "sourceAddressPrefix" : "*",
+              "destinationAddressPrefix" : "*",
+              "access" : "Allow",
+              "priority" : 1000,
+              "direction" : "Inbound"
+            }
+          },
+          {
+            "name" : "default-allow-openshift-master",
+            "properties" : {
+              "protocol" : "Tcp",
+              "sourcePortRange" : "*",
+              "destinationPortRange" : "8443",
```

Comment:
Im not sure how azure handles vips but is it possible to use 443, the console_port variable at install time, and set the vip to use 443 as well? Just to use standard ports I remember how relieved i was to move from 8443 to 443